code/src/WijmoProvider/Features/ColumnSort.ts

```diff
@@ -178,6 +178,18 @@ namespace WijmoProvider.Feature {
             );
         }
 
+        public isColumnSorted(columnID: string): boolean {
+            const column = this._grid.getColumn(columnID);
+
+            if (!column) return false;
+
+            return (
+                this._grid.provider.itemsSource.sortDescriptions.find(
```

Comment:
Maybe we should have a property on the column that should get this value (it would be easier). The way to do this is by adding this boolean value every time the OnSort event gets triggered per column.

Reply:
Agree! @glundgren93 ☝️

Reply:
The problem with this is that it is not guaranteed that the Grid will have an OnSort event. In these cases, the event will never get triggered.